cmd/gke-gcloud-auth-plugin/main.go

```diff
@@ -77,11 +81,13 @@ func execCredential() (*clientauth.ExecCredential, error) {
 }
 
 func accessToken() (string, *meta.Time, error) {
-	token, expiry, err := gcloudAccessToken()
-	if err != nil {
-		return defaultAccessToken()
+	if !*useAdcPtr {
+		token, expiry, err := gcloudAccessToken()
```

Comment:
if err != nil, then should we log.Warnings? In general I am not seeing any logging in this binary. Maybe something to consider in a follow up PR?

Reply:
kubectl will take care of informing the user of this.